src/Time/Units.hs

```diff
@@ -299,6 +301,18 @@ fortnight = time
 floorUnit :: forall time unit . (time ~ Time unit) => time -> time
 floorUnit = time . floor
 
+-- | Sums times of different units.
+--
+-- >>> minute 1 +: sec 1
+-- 61s
+--
+(+:) :: forall timeB timeA b a . (timeA ~ Time a, timeB ~ Time b, KnownDivRat a b)
+     => timeA
+     -> timeB
+     -> timeB
+t1 +: t2 = (toUnit @timeB t1) + t2
```

Comment:
`()` are redundant here. `@timeB` as well because type inference algorithm can handle this. But actually it's a good idea to keep explicit type applications here in case type of this operator will be changed in future.